classes/gateways/class.pmprogateway_stripe.php

```diff
@@ -2356,12 +2356,14 @@ private function get_price_for_product( $product_id, $amount, $cycle_period = nu
 		try {
 			$price = Stripe_Price::create( $price_args );
 			if ( ! empty( $price->id ) ) {
```

Comment:
If it doesn't make it through this statement, the function will return as null. Is it possible that `Stripe_Price::create()` could not trigger an exception and return something? Sounds edge case-y but it may be a good idea to cover that case below in `create_subscription_for_customer_from_order()` instead of just checking for `is_string()`

Reply:
Hmm, good call. I don't think reaching the end of that function should ever be possible, but accounting for it anyways would be good. Instead of returning null and having to account for it everywhere, would you rather just instead return a string `'Price could not be created'` and just have the "error" return always be a string?

Reply:
Sounds good to me, let's return a string there